src/graph.js

```diff
@@ -1,7 +1,60 @@
 /* eslint-disable */
 class Graph {
   constructor() {
-    this.graph = {};
+    this.graph = {
+      nodes: [],
+      edges: []
+    }
+  }
+  addNode(newNode, toNode = undefined) {
+    this.graph.nodes.push(newNode);
+    this.graph.edges[newNode] = [];
+    if (toNode !== undefined) {
```

Comment:
No need to say `toNode !== undefined`. You can just do `if (toNode)` to check that it is truthy.